docs/stability.md

```diff
@@ -24,6 +24,7 @@ Things considered unstable for 2.x:
 * Any feature listed as experimental or subject to change, including:
   * The [`holt_winters` PromQL function](https://github.com/prometheus/prometheus/issues/2458)
   * Remote read, remote write and the remote read endpoint
+  * Server-side HTTPS and basic authentication
```

Comment:
You should outdent this, as I'm expecting this to always be in this list.

Reply:
done